invenio_communities/subcommunities/resources/resource.py

```diff
@@ -42,3 +46,18 @@ def join(self):
             data=resource_requestctx.data,
         )
         return result.to_dict(), 201
+
+    @request_view_args
+    @response_handler(many=True)
+    @request_extra_args
+    @request_search_args
+    def search(self):
```

Comment:
nit: shouldn't this be defined by the `communities` resource instead? E.g. I want to search the subcommunities of a community.